### DISCUSSION TITLE
查询时默认添加limit如何取消

### DISCUSSION THREAD

**Discussion by a8159236** [hhyo/Archery#2791] [Q&A]
1、查询时默认添加limit行数限制，如何不自动添加limit？
![image](https://github.com/user-attachments/assets/9f639a9e-08d5-45f2-abfe-4a3874e5b069)

2、是否能够取消100、500、1000这三个选项，默认选择最大限制行数那个选项？
![企业微信截图_17255282242424](https://github.com/user-attachments/assets/1b8c5bb4-3b9e-46a2-bc5c-843f0ab803ea)

**Comment by feiazifeiazi** [comment]
你这... 需求背景是啥。  查那么多？
